spec/spec_helper.rb

```diff
@@ -25,6 +25,12 @@ module PunditSpecHelper
   config.include PunditSpecHelper
 end
 
+class NonePolicy
+end
+
+class None
+end
```

Comment:
These are only used in that one spec, so it's a shame to put them in the spec helper, but that does seem to be the approach for other tests. Would an alternate approach be to define these in a module?